src/PowerShell/Microsoft.WinGet.Configuration.Cmdlets/Cmdlets/InvokeWinGetConfigurationCmdlet.cs

```diff
@@ -0,0 +1,50 @@
+﻿// -----------------------------------------------------------------------------
+// <copyright file="InvokeWinGetConfigurationCmdlet.cs" company="Microsoft Corporation">
+//     Copyright (c) Microsoft Corporation. Licensed under the MIT License.
+// </copyright>
+// -----------------------------------------------------------------------------
+
+namespace Microsoft.WinGet.Configuration.Cmdlets
+{
+    using System.Management.Automation;
+    using System.Threading;
+    using Microsoft.WinGet.Configuration.Engine.Commands;
+    using Microsoft.WinGet.Configuration.Helpers;
+
+    /// <summary>
+    /// Invoke-WinGetConfiguration.
+    /// Start configuration and waits for completion.
+    /// </summary>
+    [Cmdlet(VerbsLifecycle.Invoke, "WinGetConfiguration")]
+    public sealed class InvokeWinGetConfigurationCmdlet : PSCmdlet
+    {
+        /// <summary>
+        /// Gets or sets the configuration file.
+        /// </summary>
+        [Parameter(
+            Mandatory = true,
+            ValueFromPipelineByPropertyName = true)]
+        public string File { get; set; }

```

Comment:
There should be a `Get-WinGetConfiguration` command that is the primary method to open a file and other commands should be taking in a file path only as a convenience.  Given that there might be an arbitrary number of mechanisms to load a configuration set, maybe we even force it to be loaded with the `Get`.